packages/mobile/locales/base/translation.json

```diff
@@ -1120,9 +1120,12 @@
     },
     "stepTwo": {
       "label": "Step 2",
-      "title": "Link a bank account",
-      "description": "You're almost done! You can connect your bank acount to Valora now via Plaid.",
-      "cta": "Coming Soon!"
+      "disabledTitle": "Link a bank account",
+      "disabledDescription": "You're almost done! You can connect your bank acount to Valora now via Plaid.",
+      "disabledCta": "Coming Soon!",
+      "title": "Enter Bank Details",
+      "description": "You're almost done! Enter bank details to link your account to Valora.",
```

Comment:
why is this separate from `disabledDescription`? They look almost identical

Reply:
I'm not sure. Its pretty consistent in the figma files that they say different things depending on wether or not its enabled

Reply:
Thats how the copy is in the figma files. I'm not sure 🤷